lte/protos/oai/nas_state.proto

```diff
@@ -475,4 +479,68 @@ message EmmContext {
   NewAttachInfo new_attach_info = 67;
   UeAdditionalSecurityCapability ue_additional_security_capability = 68;
   // TODO: add remaining emm_context elements
+
+  // 5G Guti
+  Guti_m5 m5_guti = 91;
+  Guti_m5 m5_old_guti = 92;
+  // 5G proto
+  map<uint32,SmfContext> pdu_session_id_smf_context_map =93;
+
+  }
+
+  // guti_m5_t
```

Comment:
Please move all the newly defined messages before EmmContext message definition. Definition should come before usage.

Reply:
+1 on this

Reply:
Addressed this review comment